### ISSUE TITLE
Label for CMS other non-public work

### ISSUE THREAD

**Issue by JaLuka98** [scikit-hep/mplhep#504]
According to the [CMS CAT Guidelines for figures](https://cms-analysis.docs.cern.ch/guidelines/plotting/general/#cms-label-requirements), `Other non-public work` should be labelled as either "Private work (CMS data/simulation)", "Private work (CMS data)" or "Private work (CMS simulation)" depending on the inputs for the Figure. This means that the usual mplhep functions, such as `hep.cms.label` or `hep.cms.txt` are not working for this use case because they include the bold "CMS" in front (at least I did not manage to turn it off). Is there another easy way to achieve this label or would a new function or functionality be necessary?

FYI @clelange

**Comment by andrzejnovak** [comment]
Yeah, I think this will require a new functionality. The right way probably would be a new `hep.generic.label` inheriting from https://github.com/scikit-hep/mplhep/blob/master/src/mplhep/label.py#L19 which won't have any of the relative position from ExpText adjustments.

Would you like to take a stab at this?

**Comment by JaLuka98** [comment]
> Yeah, I think this will require a new functionality. The right way probably would be a new `hep.generic.label` inheriting from https://github.com/scikit-hep/mplhep/blob/master/src/mplhep/label.py#L19 which won't have any of the relative position from ExpText adjustments.
> 
> Would you like to take a stab at this?

I can certainly give it a try, but not during the rush for ICHEP now. Let me make a note on this and try it when I have the time, otherwise I let you know. Thanks for the pointer.
